### ISSUE TITLE
Popover without `trapFocus` is not narrated by the screen reader

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#21477]
The lack of `aria` relationship between trigger and popover when `trapFocus` is not used.

* Should we care that there is no default relationship ?
* What default relationship should we add here ?
* Is this scenario just a tooltip ?

**Comment by tudorpopams** [comment]
This should be a good topic for the v-a11y team. Regarding the priority, we should put this in our backlog as medium and leave the technical decision to the team.
